### PR TITLE
Points v0

### PR DESCRIPTION
**Pull request by christianbaroni**
## What changed (plus any additional context for devs)
- Adds a points tab
- Removes animation when switching tabs
- Fixes wallet screen scroll to top on tab press

## Screen recordings / screenshots

https://github.com/rainbow-me/rainbow/assets/7061887/309b7400-a45f-4824-95d3-cc27a481cb4f

### CODE REVIEW

File: src/navigation/SwipeNavigator.js [hunk 1]
```diff
@@ -34,20 +34,22 @@ import { Box, Columns, Stack } from '@/design-system';
...
 import { discoverOpenSearchFnRef } from '@/screens/discover/components/DiscoverSearchContainer';
 import { InteractionManager, View } from 'react-native';
 import { IS_DEV, IS_IOS, IS_TEST } from '@/env';
+import config from '@/model/config';
 import SectionListScrollToTopProvider, {
   useSectionListScrollToTopContext,
 } from './SectionListScrollToTopContext';
 
-const NUMBER_OF_TABS = 3;
+const HORIZONTAL_TAB_BAR_INSET = 6;
```
Comment (skylarbarrera) [inline src/navigation/SwipeNavigator.js:50]: will need to get a look at this on android @ibrahimtaveras00 can be in release regression

**Review by skylarbarrera — APPROVED** [review]
send

**Comment by skylarbarrera** [comment]
test remote config and looks good

File: src/languages/en_US.json [hunk 2]
```diff
@@ -1228,6 +1229,10 @@
         "address_copied_to_clipboard": "Address copied to clipboard"
       }
     },
+    "points": {
+      "coming_soon_title": "Coming Soon",
+      "coming_soon_description": "Points are just around the corner.\nStart earning today."
```
Comment (walmat) [inline src/languages/en_US.json:1234]: I think this copy is a little confusing personally. What actions can the user take `today` to start earning?

File: src/navigation/SwipeNavigator.js (hunk 1 above)
Comment (walmat) [reply src/navigation/SwipeNavigator.js:50]: Looks pretty solid on my end. Tested both on a Pixel and my Nothing phone 2
Screenshot 2023-10-20 at 10 52 14 AM

**Review by walmat — APPROVED** [review]
LGTM. One small nitpick with the copy, but other than that it seems much snappier. Also seems more natural to have the tabs ordered this way imho.

**Review by dereknelson — APPROVED** [review]
merge it partner

               🤠
　 　 🤠🤠🤠
　 🤠　 🤠　 🤠
👇　 🤠　 🤠　 👇
　 　 🤠　 🤠
　 　 🤠　 🤠
　 　 👢　 👢

File: src/languages/en_US.json (hunk 2 above)
Comment (spenciefy) [reply src/languages/en_US.json:1234]: +1, what actions can I take today? 🌈

**Pull request merged**